Module/MyPythonNode.cs

```diff
@@ -129,6 +129,49 @@ public override void Execute()
 
                 host.SafeCopyToDevice();
             }
+
+            //update dashboard if needed
+            UpdateDashboard();
+        }
+
+        private Core.Dashboard.DashboardProperty GetProperty(string name)
+        {
+            return Owner.Owner.GroupedDashboard.GetByName(name);
+        }
+
+        private void UpdateDashboard()
+        {
+            if (Owner.m_DataProxy.dashboard.set != null && Owner.m_DataProxy.dashboard.set.Count > 0)
+            {
+                //for each dash-board item and its new value we want to change try to set it
+                foreach(KeyValuePair<object, object> i in Owner.m_DataProxy.dashboard.set)
+                {
+                    Core.Dashboard.DashboardProperty p = GetProperty(i.Key.ToString());
```

Comment:
This is designed to only work with grouped properties, that's maybe worth mentioning somewhere (readme?).